apps/web/pages/timetable.tsx

```diff
@@ -33,8 +33,10 @@ export default function Timetable() {
     const [loading, setLoading] = useState(true);
     const [user, authLoading, authError] = useAuthState(auth);
     const [timetable, setTimetable] = useState<Week | null>(null); // the complete timetable for the week
+    const [nextTimetable, setNextTimetable] = useState<Week | null>(null); // the complete timetable for next week
     const [current, setCurrent] = useState<number | null>(null); // current period
     const [dayView, setDayView] = useState(false); // true when displaying only one day, false when displaying the whole week
+    const [nextWeekView, setNextWeekView] = useState(false);
```

Comment:
```suggestion
    const [nextWeekView, setNextWeekView] = useState(false); // true when displaying next weeks timetable, false when displaying this week timetable
```
Maybe add a comment here?